dev/com.ibm.ws.microprofile.metrics.2.x.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/TestEnableDisableFeaturesTest.java

```diff
@@ -422,6 +422,8 @@ public void testEDF11() throws Exception {
     	serverEDF11.setMarkToEndOfLog();
     	serverEDF11.setServerConfigurationFile("server_noJDBC.xml");
     	Assert.assertNotNull("CWWKF0008I NOT FOUND",serverEDF11.waitForStringInLogUsingMark("CWWKF0008I"));
+    	//This message ID indicates the Pub/Priv Rest Handler has been initialized
+    	Assert.assertNotNull("SRVE0242I NOT FOUND",serverEDF11.waitForStringInLogUsingMark("SRVE0242I")); 
```

Comment:
Do you need to quality it with `SRVE0242I.*/metrics.*` as there might be more than one app becomes available.